tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -193,10 +259,12 @@ def insert_agent(agent_id, agent_name='TestName'):
 
 
 def remove_agent(agent_id):
-    """Function that wraps the needed queries to remove an agent.
+    '''
+    Function that wraps the needed queries to remove an agent.
+
     Args:
-        agent_id(int): Unique identifier of an agent
-    """
+        agent_id(int): Unique identifier of an agent.
+    '''
```

Comment:
I would not insert a new line before the one-line summary and use `"""` instead.

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)